lms/djangoapps/course_home_api/progress/v1/serializers.py

```diff
@@ -0,0 +1,15 @@
+# pylint: disable=abstract-method
+"""
+Progress Tab Serializers
+"""
+
+from rest_framework import serializers
+from lms.djangoapps.course_home_api.outline.v1.serializers import CourseBlockSerializer
+
+
+class ProgressTabSerializer(serializers.Serializer):
+    """
+    Serializer
```

Comment:
You can update this later, but just want to call out that we should add in some more details here.